runtime/compiler/codegen/CodeGenGC.cpp

```diff
@@ -266,6 +268,24 @@ J9::CodeGenerator::createStackAtlas()
          }
       }
 
+   // Iniialize colour mapping
+   //
+   int32_t *colourToGCIndexMap = 0;
```

Comment:
Is this related to colouring RA? I think we deprecated that. I'd like to understand more about this.

Reply:
No, this is for the CompactLocals optimization.  It uses graph colouring to reduce the stack space requirements by sharing storage for non-interfering locals.